src/content/docs/hello-there.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Gdzie zacząć?
+description: Pierwsze kroki w dokumentacji Solvro
+---
+
+## Na początek
+
+### Co można tu znaleźć?
+
+Nasza dokumentacja to źródło wiedzy o naszym kole. Znajdziesz tutaj:
+
+- Informacje o technologiach, które wykorzystujemy, oraz poradniki, jak z nich korzystać.
+- Szczegóły dotyczące celów naszego koła.
+- Informacje o zarządzie i składzie członków.
+- Opisy realizowanych projektów.
+- Wskazówki dotyczące narzędzi używanych przez koło.
+- Wszystko, co potrzebne, aby efektywnie działać w naszym kole i lepiej je zrozumieć.
+
+Ten dokument jest punktem startowym dla każdego, kto odwiedza naszą dokumentację.
+
+### Jak używać tej dokumentacji?
+
+Dokumentacja jest podzielona na sekcje:
+
+1. Start here - wszystko co potrzebne do korzystania z dokumentacji.
+2. Handbooki - poradniki od członków, dla członków. Dotyczą technologii, narzędzi, organizacji pracy oraz umiejętności miękkich.
+3. Projekty - informacje o projektach nad którymi działamy.
+
+Skorzystaj z menu nawigacyjnego po lewej stronie, aby przejść do interesujących Cię obszarów.
+
+### Jak dołożyć cegiełkę do dokumentacji?
+
+Nasza dokumentacja jest otwarta na wszelkie sugestie. Widzisz błąd? Chcesz coś zmienić lub dodać? A może masz pomysł na nowy materiał? Nie czekaj! Wszystkie potrzebne informacje znajdziesz [tutaj](https://docs.solvro.pl/how-to-write-docs).
+
+## Przewodnik po dokumentacji
+
+Znając [wstępne informacje](https://docs.solvro.pl/solvro/hello-there/#na-początek) o naszej dokumentacji, chcielibyśmy zaproponować Ci kolejność w której powinieneś przejrzeć nasze zasoby.
+
+### Dla gości
+
+- [Poznaj nasz zarząd](https://docs.solvro.pl/solvro/vii_board/)
+- [Zapoznaj się z naszą strategią](https://docs.solvro.pl/solvro/strategy/)
+
+### Dla nowych członków
+
+Ogólne informacje:
+
+- [Poznaj nasz zarząd](https://docs.solvro.pl/solvro/vii_board/)
```

Comment:
mam wrażenie że zarząd to jedna z mniej ważnych informacji o kole i nie powinna być na początku

Reply:
```suggestion
- [Poznaj nasz zarząd](https://docs.solvro.pl/solvro/vii_board/)
- [Zapoznaj się z naszą strategią](https://docs.solvro.pl/solvro/strategy/)

### Dla nowych członków

Ogólne informacje:

```